packages/kbn-check-saved-objects-cli/src/snapshots/compare.ts

```diff
@@ -112,9 +131,24 @@ function getMutatedModelVersions(
   infoBefore: MigrationInfoRecord,
   infoAfter: MigrationInfoRecord
 ): string[] {
-  const mutatedModelVersions = infoBefore.modelVersions.filter(
-    (summaryBefore, index) => !equal(summaryBefore, infoAfter.modelVersions[index])
-  );
+  const mutatedModelVersions = infoBefore.modelVersions.filter((summaryBefore, index) => {
+    const summaryAfter = infoAfter.modelVersions[index];
+
+    if (!summaryBefore.modelVersionHash) {
+      // TODO remove this conditional when all the baseline snapshots have the new hash properties
+      // we're comparing against an old snapshot, downgrade the infoAfter one to match the old format
+      const to: Partial<ModelVersionSummary> = cloneDeep(summaryAfter);
+      delete to.modelVersionHash;
+      // @ts-ignore we're simulating an older version of the type without the new properties
```

Comment:
I started with `DeepPartial<ModelVersionSummary>` to handle removing required properties   which was far more verbose than simply ignoring for now.

Once the snapshots have built up over time and have the new hashes, we'll be deleting the conditional mutations and won't need to ignore type-issues.